src/instance.jl

```diff
@@ -8,8 +8,7 @@ function nametest(instance::MOI.AbstractInstance)
         @test MOI.get(instance, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64},MOI.LessThan{Float64}}()) == 0
 
         v = MOI.addvariables!(instance, 2)
-        @test MOI.canset(instance, MOI.VariableName(), v[1])
-        @test MOI.canget(instance, MOI.VariableName(), v[1])
```

Comment:
why remove canget ?

Reply:
The canset should maybe be moved 2 lines below just before the `set!`

Reply:
Done